client/src/assets/scss/app.scss

```diff
@@ -0,0 +1,38 @@
+@import "./foundation/settings";
+@import "~foundation-sites/scss/foundation";
+@import "./foundation/variables"; 
```

Comment:
We would then want to update this import statement after moving the color variables file

Reply:
Updated!